### ISSUE TITLE
[Spike] Refactor updater tests

### ISSUE THREAD

**Issue by creviera** [freedomofpress/securedrop-workstation#785]
# Description

There is some confusion around how the launcher tests work because of how we try to automatically detect OS version and support both 4.0 and 4.1 as well as pyqt4 and pyqt5 and bullseye ad buster... let's look into simplifying and removing test support for stuff we don't need to support anymore after August 4th.

It might help to explain what it is we are actually testing as well (I think just the GUI, right?).

**Comment by rocodes** [comment]
Yes,  the pyqt4/5 dual support is just for testing the updater GUI inside dom0 (pyqt4 for 4.0/fedora-25) vs outside dom0, so there's a lot we can remove now that we're on 4.1.

**Comment by zenmonkeykstop** [comment]
This should be revisited to see if we can drop further stuff now we're 4.2 only.

**Comment by zenmonkeykstop** [comment]
Keeping for now - updater tests can be 4.2-only etc as we don't support older versions of Qubes.
